packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -224,13 +216,13 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
           {
             backgroundColor: globalTokens.color.grey[14],
```

Comment:
Background color can be mapped to neutralForeground1

Reply:
yeah, the only question I had if this was expected. And I still didn't get an answer. So I'll update as on readlines